services/booking-api/lambdas/cancel.js

```diff
@@ -0,0 +1,18 @@
+import { throwError } from '@helsingborg-stad/npm-api-error-handling';
+import to from 'await-to-js';
+
+import * as response from '../../../libs/response';
+
+import booking from '../helpers/booking';
+
+export async function main(event) {
+  const bookingId = event.pathParameters.id;
+  const body = { bookingId };
+
+  const [error] = await to(booking.cancel(body));
+  if (error) {
+    throwError(error);
```

Comment:
throwError expects two args, statusCode and message. If error is the status code then this is fine I guess? :)

Reply:
Not entirely correct. throwError expects 2 arguments but the second one is optional since it will receive a default value if no message is added.